lexer/src/token.rs

```diff
@@ -148,6 +146,13 @@ pub enum TokenType {
 
 impl TokenType {
 
+    ///is this lexeme a keyword of the language ?
+    pub fn is_keyword(self) -> bool {
+        matches!(self,
+            Fun | Use | IntLiteral | Else | For | In | While | Match | Val | Var
```

Comment:
I don't think that int literals are keywords.

Reply:
fail